plugins/innogrit/innogrit-nvme.c

```diff
@@ -146,11 +146,13 @@ static int nvme_vucmd(int fd, unsigned char opcode, unsigned int cdw12,
 
 	memset(&cmd, 0, sizeof(cmd));
 	cmd.opcode = opcode;
+	cmd.cdw2  = 0x69677673;
```

Comment:
Usually, it's a good practice to use a define for a magic number.

Reply:
I have modify to add a define for magic number